src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+
+Displays is a transitive verb, meaning that the action transfers to something
+else. For example, a screen displays an image. While displays requires a direct
+object, appears does not. For example, an image appears on screen.
+
+### as
+
+Avoid using as as a synonym for because or while. While grammatically correct,
+these uses make reading more difficult for a global audience.
+
+- Compare with [because](/content/word-list#because)
+
+<BestPractice>
+  <Do>
+    <p>Use the active voice whenever possible because it’s easier to translate.</p>
+  </Do>
+  <Dont>
+    <p>Use the active voice whenever possible as it is easier to translate.</p>
+  </Dont>
+</BestPractice>
+
+### because, since
+
+Use because instead of since. The reason is that since can cause problems with
+machine translation. Use because to refer to the reason for something, and since
+to refer to a passage of time.
+
+### between
+
+Use between when referring to 2 persons (or things) or when referring to
+relationships between two items at a time, regardless of the total number of
+items.
+
+- Compare with [among](/content/word-list#among)
+
+<BestPractice>
+  <Do>
+    <p>Move between the two tickets at the top of the list.</p>
+  </Do>
+  <Dont>
+    <p>Move between several tickets on the page.</p>
+  </Dont>
+</BestPractice>
+
+### blacklist, whitelist
+
+Avoid this usage. Use blocked or allowed instead.
+
+### CC, CCs, CCed
+
+In the product, use uppercase. OK to use as a noun ("the CC", "the CCs"), as a
+transitive verb ("CC me"), or as an adjective ("the CCed agent").
+
+In documentation, it’s permitted to use lowercase and apostrophes (cc, cc’s,
+cc’ed).
+
+### channel
+
+Use lowercase for general references. Capitalize when referring to t`he Channel
+framework or describing features of the framework, such as the Channel pull
+endpoint.
+
+### chat, Zendesk Chat
+
+Lowercase when talking about a chat message or the verb. Capitalize when
+referring to the feature or the product, such as Zendesk Chat.
+
+### choose, select
+
+Use choose when the user must make a decision. Use select when the user is
+presented with an item from a list to carry out a decision already made.
+
+### click, swipe, tap
+
+Avoid using platform-specific words like click, swipe, or tap.
+
+Use click to describe the action of pressing a key, mouse, or button. For mobile
+or screen-based physical contact, use tap. Pair click directly with the button
+or menu label without “on” or “at.”
+
+<BestPractice>
+  <Do>
+    <p>Preview document</p>
+  </Do>
+  <Dont>
+    <p>Click on preview</p>
+  </Dont>
+</BestPractice>
+
+### Customer Support
+
+Use Zendesk Customer Support or Customer Support when referring to the place
+Zendesk customers go for support on our product. Avoid Support team.
+
+### CSAT
+
+All caps. Short for customer satisfaction. Use only as an adjective, as in CSAT
+rating or CSAT survey.
+
+### data
+
+Always use data with a singular verb. In other words, “the data is...”, “the
```

Comment:
Always use _data_ with a singular verb. In other words, “The data is...”, “the
data goes...” whether you mean a collection of facts (plural) or information
(singular). If you want to emphasize that something is plural, rewrite to use a
term such as facts or numbers.